ImagePicker/UICollectionView+CellRegistrator.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  UICollectionView+CellRegistrator.swift
+//  ImagePicker
+//
+//  Created by Anna Shirokova on 14/06/2018.
```

Comment:
we should keep the headers as following 
// Copyright © 2018 INLOOPX. All rights reserved.